app/Infrastructure/Http/Requests/CreateVacationRequestRequest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Toby\Infrastructure\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Toby\Eloquent\Models\VacationRequest;
+
+class CreateVacationRequestRequest extends FormRequest
```

Comment:
Honestly, I would prefer to not use form request for things like that (optional GET params,  authorization, etc), at least not in a project where it is not standard and we did other authorization inside controller methods.

Reply:
Restore this version?

https://github.com/blumilksoftware/toby/commit/5f02e23f511a4381199f6b9d9e38f6cc4da638c1#diff-0a0b7b715da129fe7e856b493c478b10a56016e20690ffdc592c20239b4a8e7d

Reply:
Yeah, but I don't think you need to find user by id in this case. Just pass user Id from request to Inertia as you did with start date.